### ISSUE TITLE
(Home/Linux)brew formula to easy install on Mac/Linux with Mono

### ISSUE THREAD

**Issue by diegobfernandez** [urbas/bud#2]
As an equivalent of chocolatey it would be a great choice.

**Comment by urbas** [comment]
Yup, this definitely needs to be done. Although I'll probably create a DEB package for Ubuntu rather than brew.

Also, I don't have a mac. Could I ask you to test the packages when I manage to prepare the formula?

**Comment by diegobfernandez** [comment]
I'm currently on Fedora and I really prefer brew, altough deb for ubuntu is great.
But I can get a mac to test on.
Let me know when it's ready.

**Comment by urbas** [comment]
Here's the plan:
- I will start with Ubuntu (just because I have Ubuntu already installed).
- Then I'll prepare a package for Fedora (have to get a VM going).
- And then a formula for Mac.
 
I'll create branches and link them to this issue. I'll probably start it this weekend.

**Comment by urbas** [comment]
I started working on the Ubuntu package. You can track progress on this branch: https://github.com/urbas/bud/tree/feature/2-ubuntu-package

I managed (with great pain :) ) to build up the package. I'm still testing it and fixing last bits though.

Now I have to find a way to distribute it too.

**Comment by urbas** [comment]
Alright, Bud now knows how to build its own Ubuntu package.

I introduced a macro in Bud's build which builds the package on Windows too (no need to do it in Ubuntu or Debian).

Remaining problem: a way to distribute the package.

**Comment by urbas** [comment]
The README.md file now lists the Ubuntu DEB package.

Next step: prepare an OS X brew formula, distribute it, and add a link to it in README.md.

**Comment by urbas** [comment]
I have restarted bud, therefore this issue has been set back for a while.

I still intend to address this issue.

In the meantime, I have added installation instructions to the [README.md file](https://github.com/urbas/bud/blob/master/README.md#installation).
